images/taint-manager/taintmanager.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+    "context"
+    "flag"
+    "fmt"
+    v1 "k8s.io/api/core/v1"
+    "k8s.io/apimachinery/pkg/api/errors"
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    "k8s.io/apimachinery/pkg/util/validation"
+    "k8s.io/client-go/kubernetes"
+    "k8s.io/client-go/rest"
+    "log"
+    "os"
+    "strings"
+)
+
+func main() {
+    taintAdd := flag.String("add", "", "The taint to add")
+    taintRemove := flag.String("remove", "", "The taint to remove")
+    flag.Parse()
+    tAdd, errAdd := parseTaint(*taintAdd)
+    tRemove, errRemove := parseTaint(*taintRemove)
+
+    if errAdd != nil && errRemove != nil {
+        log.Println("Please specify at least one option -add or -remove")
+        log.Println("Usage: taintmanager [-add TAINT] [-remove TAINT]")
+        flag.PrintDefaults()
+        os.Exit(1)
+    }
+
+    // creates the in-cluster config
+    config, err := rest.InClusterConfig()
```

Comment:
Can you fallback to kubeconfig here if inclusterconfig is not available? That would allow us to test this locally *much* more easily, and get rid of all the extra YAML bits and tiltfile required to simply test it.

Reply:
Good idea. out-cluster kubeconfig and node name parameter are added.